collectors/bzimport/convertors.py

```diff
@@ -654,7 +654,7 @@ def get_meta_attr(self, cve_id):
         meta_attr["alias"] = self.alias
         meta_attr["depends_on"] = self.depends_on
         meta_attr["related_cves"] = [c for c in self.cve_ids if c != cve_id]
-        meta_attr["bz_summary"] = self.flaw_bug["summary"]
+        meta_attr["bz_summary"] = self.flaw_bug["cve_description"]
```

Comment:
`self.flaw_bug` contains data in Bugzilla format, so `summary` should stay here (Bugzilla cannot recognize `cve_description`)